src/App.vue

```diff
@@ -247,14 +248,15 @@ import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
     WebChat,
     ...Dialogs,
     ...Views
-  }
+  },
+  mixins: [
+    CommonMixin,
+    DateMixin,
+    FilingTemplateMixin,
+    NameRequestMixin
+  ]
 })
-export default class App extends Mixins(
-  CommonMixin,
-  DateMixin,
-  FilingTemplateMixin,
-  NameRequestMixin
-) {
+export default class App extends Vue {
```

Comment:
The reasons for doing this are manifold:
- Vue 2.7 (and future Vue 3) don't like the special decorators, esp. `Mixins`
- the VS Code Volar (Vue Language Features) plugin only intermittently recognizes mixins used the old way; often it would show errors on mixin functions
- in some cases, you can't extend more than 1 mixin at a time; this bypasses that

In a nutshell, the new code is better for the future.